src/CtaLink/CtaLink.js

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ctaLinkIcons, ctaLinkDisplay, ctaLinkColors, ctaLinkAnimations, ctaLinkButtonSizes, ctaLinkButtonVariants } from './CtaLink.levers';
+import Icon from 'react-hero-icon';
+import clsxd from 'clsx-dedupe';
+
+/**
+ * CTA Link Component
+ *
+ */
+export const CtaLink = ({ classes = {}, text, srText, display, link, color, isButton, variant, size, icon, animate, ref, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const levers = {};
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  // display
+  if (display && ctaLinkDisplay.includes(display)) {
+    switch (display) {
+      case 'inline-block':
+        levers.display = 'su-inline-block';
+        break;
+
+      case 'block':
+        levers.display = 'su-block';
+        break;
+
+      case 'flex':
+        levers.display = 'su-flex';
+        break;
+    }
+  }
+
+  // color
+  if (color && ctaLinkColors.includes(color) && !isButton) {
+    switch (color) {
+      case 'red':
+        levers.color = 'su-text-digital-red hocus:su-text-black';
+        break;
+
+      case 'blue':
+        levers.color = 'su-text-digital-blue hocus:su-text-black';
+        break;
+
+      case 'white':
+        levers.color = 'su-text-white hocus:su-text-white';
+        break;
+    }
+  }
+
+  // isButton
+  if (isButton) {
+    levers.isButton = 'su-cta-btn su-font-regular su-leading-none';
+  }
+  else {
+    levers.isButton = 'su-cta-link su-text-19 md:su-text-20';
+  }
+
+  // variant
+  if (variant && ctaLinkButtonVariants.includes(variant) && isButton) {
```

Comment:
So the variant prop only works if the link is a button? I'd like to see props not rely or depend on another prop setting in order to work. This could lead to bad DX. What happens if you allow for links and buttons to have variants?

Reply:
This is related to the answer right below this one - let me know what you think.

Reply:
I'm thinking about adding a "none" or "unset" option to the button variants, so it's easier when someone want to pass custom TW classes for colors and they won't get overridden by the color classes we added for the button variants.

And, would it help if I rename this option `buttonVariants` or `buttonStyles`? I know that's still not ideal - let me know if you rather we have 2 separate files for links and link buttons.